tslint.js

```diff
@@ -35,6 +35,13 @@ module.exports = {
       "check-open-brace",
       "check-whitespace"
     ],
+    "ordered-imports": [
```

Comment:
@brandyscarney note that with the latest changes in TSLint you now use:
```json
"ordered-imports": {
    "option":{
        "import-sources-order": "any",
        "named-imports-order": "lowercase-last"
    }
}
```

That's what my prev. PR address, but it hasn't been merged yet.